core/src/main/scala/dagr/core/cmdline/parsing/DagrCommandLineParser.scala

```diff
@@ -312,26 +324,27 @@ private[cmdline] class DagrCommandLineParser(val commandLineName: String, val in
       if (name == command) {
         throw new CommandLineParserInternalException("Command matches when searching for the unknown: " + command)
       }
-      val distance: Int = if (name.startsWith(command) || (MINIMUM_SUBSTRING_LENGTH <= command.length && name.contains(command))) {
+      val distance: Int = if (name.startsWith(command) || (MinimumSubstringLength <= command.length && name.contains(command))) {
         0
       }
       else {
         levenshteinDistance(command, name, 0, 2, 1, 4)
       }
       distances.put(clazz, distance)
-      if (distance < bestDistance) {
-        bestDistance = distance
-        bestN = 1
-      }
-      else if (distance == bestDistance) {
-        bestN += 1
+      distance match {
```

Comment:
For simple boolean checks, I find this pattern of using matches with a bunch of `case x if x ....` to be more verbose, and less clear.